src/parsers/json/Parser.h

```diff
@@ -86,11 +86,11 @@ class Parser : public SEMBA::Parsers::Parser {
     static bool checkExtendedVersionCompatibility(const std::string& version);
 
     static const Geometry::ElemR* boxToElemGroup(
-            Geometry::Mesh::Geometric& mesh,
+            Geometry::Mesh::Unstructured& mesh,
```

Comment:
This is much better as you have written it... no need for geometric, just unstructured.